src/nvim/api/private/helpers.h

```diff
@@ -81,7 +81,7 @@
 #define api_init_float
 #define api_init_string = STRING_INIT
 #define api_init_buffer
-#define api_init_window
+#define api_init_nvimwindow
```

Comment:
I would not bother changing these helper macros unless we plan to add "nvim" prefix to all of them (`api_init_string` => `api_init_nvimstring`, ...). But that's probably unnecessary. At least wait until we have a reason to do it (e.g. build conflicts).